File: Perl Projects/MassiveInput.pl
my($linesToRead, $divisor) = split(/ /, <>);

$count = 0;
for($line = 0; $line < $linesToRead; ++$line)
{
  $currentValue = <>;
  if($currentValue % $divisor == 0)
  {   
    ++$count;  
  }
}

print "$count\n";

File: Python Projects/MassiveInput.py
import fileinput

lineCount = 0
count = 0
divisor = 1
for line in fileinput.input():
  
  if lineCount == 0:
    divisor = line.split()[1]
  else:
    if int(line) % int(divisor) == 0:
     count = count + 1
  
  lineCount = lineCount + 1

print str(count)